file_paths.md

```diff
@@ -121,14 +126,22 @@ The files annotated by an '*' indicate that they are created by package installa
         ssl *
         VERSION                           # puppet-agent package version
 
+    /opt/puppetlabs/pxp-agent *
+        modules *
+            puppet *                      # executes Puppet and provides results of Puppet runs
```

Comment:
@kylog This one isn't prohibitively sticky, but it does require renaming and testing in pxp-puppet and the mock agent. Not a big deal, just something we need to be aware of.